Makefile

```diff
@@ -742,15 +742,22 @@ out/doc/api/assets/%: doc/api_assets/% out/doc/api/assets
 run-npm-ci = $(PWD)/$(NPM) ci
 
 LINK_DATA = out/doc/apilinks.json
+VERSIONS_DATA = out/doc/previous-versions.json
 gen-api = tools/doc/generate.js --node-version=$(FULLVERSION) \
-		--apilinks=$(LINK_DATA) $< --output-directory=out/doc/api
+		--apilinks=$(LINK_DATA) $< --output-directory=out/doc/api \
+                --versions-file=$(VERSIONS_DATA)
```

Comment:
indent inconsistency I think, shouldn't this line be at the same level as the previous?

Reply:
Yes, tabs vs space issue. Fixed.